.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
Should we also include Media/SDK version in the bug report template?

Reply:
Yes I will add it